testsuite/Gemfile

```
@@ -17,7 +17,7 @@ gem 'minitest'
 gem 'websocket'
 gem 'websocket-driver'
 gem 'syntax'
-gem 'parallel_tests'
+gem 'parallel_tests', '3.1.0'
```

```suggestion
gem 'parallel_tests', '3.11.1'
```
I guess we can now use the most recent version.